app/controllers/index.py

```diff
@@ -47,19 +51,53 @@ async def communities():
     return render_response('communities.jinja2', {'local_chapters': LOCAL_CHAPTERS})
 
 
+@router.get('/copyright/{locale:str}')
+async def copyright_i18n(locale: str):
+    if not is_valid_locale(locale):
+        return Response(None, status.HTTP_404_NOT_FOUND)
+    with translation_context(primary_translation_language()):
+        copyright_notice_title = t('site.copyright.foreign.title')
+        original_copyright_link_text = t('site.copyright.foreign.english_link')
+        original_copyright_link = f'<a href="/copyright/en">{original_copyright_link_text}</a>'
+        copyright_notice_html = t('site.copyright.foreign.html', english_original_link=original_copyright_link)
+    with translation_context(locale):
+        copyright_translated_title = t('site.copyright.legal_babble.title_html')
+        copyright_content = render('copyright_content.jinja2', **_get_default_data())
+    should_show_notice = locale != primary_translation_language() or primary_translation_language() != DEFAULT_LANGUAGE
+    return render_response(
+        'copyright.jinja2',
+        dict(
+            copyright_content=copyright_content,
+            should_show_notice=should_show_notice,
+            copyright_notice_title=copyright_notice_title,
+            copyright_notice_html=copyright_notice_html,
+            copyright_translated_title=copyright_translated_title,
+        ),
```

Comment:
{'should_show_notice': ...} for consistency? dict() works but seems unnecessary. Similarly in the about controller.